From 79e364c3050202da916e2e97a2ed6bca65f3d519 Mon Sep 17 00:00:00 2001
From: Madhu <enometh@net.meer>
Date: Thu, 19 Nov 2020 11:13:12 +0530
Subject: [PATCH 4/5] src/profile-editor.c: (profile_prefs_load): choose face

calling gtk_font_chooser_set_level gives wrong results. choosing Luxi
Mono actually chooses Luxi Mono Bold Oblique - and doesn't let you
choose Luxi Mono Regular. Likewise Consolas Bold is chosen by default.

So allow selecting the style. And everything else
---
 src/profile-editor.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/profile-editor.cc b/src/profile-editor.cc
index 7357ec4..35a64b9 100644
--- a/src/profile-editor.cc
+++ b/src/profile-editor.cc
@@ -1329,6 +1329,7 @@ profile_prefs_load (const char *uuid, GSettings *profile)
 #if GTK_CHECK_VERSION (3, 24, 0)
   gtk_font_chooser_set_level (GTK_FONT_CHOOSER (w),
 			      GtkFontChooserLevel(GTK_FONT_CHOOSER_LEVEL_FAMILY |
+	      GTK_FONT_CHOOSER_LEVEL_STYLE | GTK_FONT_CHOOSER_LEVEL_VARIATIONS | GTK_FONT_CHOOSER_LEVEL_FEATURES |
 						  GTK_FONT_CHOOSER_LEVEL_SIZE));
 #endif
 
-- 
2.39.2.101.g768bb238c4

